src/get_family_names.py

```diff
@@ -70,13 +72,23 @@ def flatten_parents(df):
 
 
 def get_all_congress_people_ids():
+    print('Fetching all congresspeople ids', end='\r')
     ids_series = [read_csv(name)['congressperson_id']
                   for name in ['current-year', 'last-year', 'previous-years']]
     return list(pd.concat(ids_series).unique())
 
 
+def find_latest_date():
+    date_regex = re.compile('\d{4}-\d{2}-\d{2}')
+
+    matches = (date_regex.findall(f) for f in os.listdir(DATA_DIR))
+    dates = (l[0] for l in matches if l)
```

Comment:
Makes sense. I'll have a look at how the other scripts are doing it (I had only looked at the `fetch_cnpj_info`) and use your suggestions to come up with a good way to load files.